kubernetes/template-generator/template_generator.py

```diff
@@ -0,0 +1,358 @@
+#!/usr/bin/env python
+
+import re
+import yaml
+import json
+
+from container_os import *
+
+### TEMPLATE GENERATORS
+class SanitizeException(Exception):
+    pass
+
+class TemplateGenerator(object):
+    """
```

Comment:
this is very strange doctrines, are those class or function? if function afaik those should be in the function body not bellow or above.